system/lib/libc/musl/src/unistd/dup3.c

```diff
@@ -15,9 +15,6 @@ int __dup3(int old, int new, int flags)
 		if (r!=-ENOSYS) return __syscall_ret(r);
 	}
 	while ((r=__syscall(SYS_dup2, old, new))==-EBUSY);
-#ifndef __EMSCRIPTEN__ // CLOEXEC makes no sense for a single process
-	if (flags & O_CLOEXEC) __syscall(SYS_fcntl, new, F_SETFD, FD_CLOEXEC);
-#endif
```

Comment:
I think by removing this chunk of code you are removing not just the new ifdef lines we added for emscripten but also the original musl line we wanted to avoid? That might make upgrading musl harder later.

Reply:
oops, good catch.  Reverting.